.github/workflows/ci.yml

```diff
@@ -8,6 +8,16 @@ on:
 
 env:
   RUST_BACKTRACE: 1
+  DOCSRS_PREFIX: ignored/cratesfyi-prefix
+  DOCSRS_DATABASE_URL: postgresql://cratesfyi:password@localhost:15432
+  DOCSRS_LOG: docs_rs=debug,rustwide=info
+  AWS_ACCESS_KEY_ID: cratesfyi
+  AWS_SECRET_ACCESS_KEY: secret_key
+  S3_ENDPOINT: http://localhost:9000
+  DOCSRS_INCLUDE_DEFAULT_TARGETS: false
+  DOCSRS_DOCKER_IMAGE: ghcr.io/rust-lang/crates-build-env/linux-micro
+  SENTRY_ENVIRONMENT: dev
+  DOCSRS_METADATA_HOST_TARGET: aarch64-unknown-linux-gnu
```

Comment:
This shouldn't be necessary, it's set by the `build.rs`.

Reply:
@Nemo157 I removed the line